src/components/views/context_menus/MessageContextMenu.js

```diff
@@ -88,6 +88,14 @@ module.exports = React.createClass({
         if (this.props.onFinished) this.props.onFinished();
     },
 
+    onForwardClick: function() {
+        dis.dispatch({
+            action: 'forward_message',
+            content: this.props.mxEvent.getContent(),
```

Comment:
hm, what about event type?  what happens if you try to forward a non m.room.message?  I suspect we should forbid forwarding of state events(!)